src/components/Proposal/VoteBox.tsx

```diff
@@ -10,6 +10,7 @@ import * as css from "./Proposal.scss";
 
 interface IProps {
   currentVote: number;
+  voterReputation: number;
```

Comment:
again currentAccountReputation feels bit more clear to me

Reply:
updated: https://github.com/daostack/alchemy/pull/39/commits/b8dd04d7e8e23e73e18aeadf3395fec6333d802a